### ISSUE TITLE
Longform option `--file` argument not recognized without `=`

### ISSUE THREAD

**Issue by jasonkarns** [github/hub#1201]
Per the [source documentation](https://github.com/github/hub/blob/master/commands/release.go#L69-L70):

> -F, --file <FILE>
>   Read the release title and description from <FILE>.

I would have expected `hub release create --draft --file - "v2.3.2"` to take the file contents from STDIN. However, I received:

```
flag needs an argument: --file
usage: git release create [-d] [-p] [-a <ASSETS_FILE>] [-m <MESSAGE>|-f <FILE>] <TAG>

Creates a new release in GitHub for the project that the "origin" remote points to.
It requires the name of the tag to release as a first argument.

Specify the assets to include in the release via "-a".

Without <MESSAGE> or <FILE>, a text editor will open in which title and body
of the release can be entered in the same manner as git commit message.

If "-d" is given, it creates a draft release.

If "-p" is given, it creates a pre-release.
```

Apparently, it needed the equal sign, because this variant worked: `hub release create --draft --file=- "v2.3.2"`

I would expect the longform options to accept their args without the `=`, as it is shown in the docs.

```
$ hub --version
git version 2.9.0
hub version 2.2.3
```

**Comment by mislav** [comment]
Thanks for the report! It looks like https://github.com/ogier/pflag project only supports long-form flags with equal signs. I think the docs need to be amended to reflect that.

**Comment by mislav** [comment]
The documentation was fixed in #1971

Tracking feature change in #1961
